lib/presentation/widgets/m3_widgets/cards/outlined_trip_card.dart

```diff
@@ -106,27 +108,28 @@ class OutlinedTripCard extends StatelessWidget {
               ],
             ),
           ),
-          const Divider(),
-          Padding(
-            padding: const EdgeInsets.symmetric(
-              horizontal: AppDimensions.d24,
-              vertical: AppDimensions.d10,
+          if (withActionButtons ?? true) const Divider(),
+          if (withActionButtons ?? false)
```

Comment:
```suggestion
          if (!withActionButtons)
```

Reply:
If the condition is the same you can add the buttons and the divider in the list(`[ ]`) with one condition